PROVISIONING-METAL.md

```diff
@@ -92,9 +92,22 @@ Interface names are expected to be correct as per `udevd` naming, no interface n
   * `enabled` (boolean, required): Enables DHCP6.
   * `optional` (boolean): the system will request a lease using this protocol, but will not wait for a valid lease to consider this interface configured.
 
+As of version `2` static addressing with simple routes is supported via the below settings.
+Please keep in mind that when using static addresses, DNS information must be supplied to the system via user data: [`settings.dns`](https://github.com/bottlerocket-os/bottlerocket#network-settings).
+* `static4` (map): IPv4 static address settings.
+  * `addresses` (list of quoted IPv4 address including prefix): The desired IPv4 IP addresses, including prefix i.e. `["192.168.14.2/24"]`.  The first IP in the list will be used as the primary IP which `kubelet` will use when joining the cluster.  If ipv4 and ipv6 static addresses exist, the first ipv4 address is used.
+* `static6` (map): IPv6 static address settings.
+  * `addresses` (list of quoted IPv6 address including prefix): The desired IPv6 IP addresses, including prefix i.e. `["2001:dead:beef::2/64"]`.  The first IP in the list will be used as the primary IP which `kubelet` will use when joining the cluster.  If ipv4 and ipv6 static addresses exist, the first ipv4 address is used.
+
```

Comment:
nit: for consistency, use "IPv4" and "IPv6" throughout, rather than also the lowercase forms